### PR TITLE
Add configuration to support two factor authentication

### PR DESCRIPTION
**Pull request by zzuzzy**
Add support to two factor authentication

### CODE REVIEW

File: is_core/site.py [hunk 1]
```diff
@@ -88,6 +88,12 @@ def get_urls(self):
                                   auth_resource_class)
             urlpatterns += urls_wrapper(pattern.get_url())
 
+        if settings.AUTH_CODE_CHECK_LOGIN_VIEW:
+            urlpatterns += urls_wrapper(
+                url(r'^{}$'.format(settings.CODE_CHECK_URL[1:]),
+                    str_to_class(settings.AUTH_CODE_CHECK_LOGIN_VIEW).as_view(), name='codechecklogin'),
```
Comment (matllubos) [inline is_core/site.py]: code-validation-login

File: is_core/config.py [hunk 2]
```diff
@@ -1,7 +1,20 @@
...
 
 
+def _get_auth_login_view():
+    if 'auth_token' in django_settings.INSTALLED_APPS and django_settings.AUTH_TOKEN_TWO_FACTOR_ENABLED:
+        return 'auth_token.contrib.is_core_auth.views.TwoFactorLoginView'
+    elif 'auth_token' in django_settings.INSTALLED_APPS:
+        return 'auth_token.contrib.is_core_auth.views.LoginView'
+    else:
+        return 'is_core.views.auth.LoginView'
+
```
Comment (matllubos) [inline is_core/config.py]: missing newline

File: is_core/config.py [hunk 3]
```diff
@@ -1,7 +1,20 @@
 from django.conf import settings as django_settings
 
 
+def _get_auth_login_view():
```
Comment (Formulka) [inline is_core/config.py]: Why is this being inferred like this and is not in the Django settings directly? Applies to other parts of the configuration as well.

Comment (rubickcz) [reply is_core/config.py]: I think it's because of easier configuration, you don't need to do too much by hand.

**Pull request merged**